src/reducers/initialState.js

```diff
@@ -0,0 +1,7 @@
+const initialState = [
+  { id: 0, title: 'Friday 13', category: 'Horror' },
+  { id: 1, title: 'Wild-life Dangers', category: 'Learning' },
+  { id: 2, title: 'Toonami', category: 'Kids' },
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Friday 13', category: 'Horror'}
```